firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +137,71 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+
+    def test_box(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.box)
```

Comment:
For all those tests please check not for notNone but for the exact type of element like: 'self.assertEqual('textbox', urlbar.get_attribute('localName'))'. Also you can put all in a single line, we have a limit of 100 chars per line in this repository. The same applies to all the other test methods in this file.